patch -p0 < /var/www/html/argentina/profiles/argentinagobar/modules/contrib/ctools/includes/content.inc 
--- modules/contrib/ctools/includes/content.inc 2017-09-26 12:05:24.198844506 -0300
+++ modules/contrib/ctools/includes/content.inc.modificado  2017-09-26 12:05:29.839058737 -0300
@@ -491,16 +491,25 @@ function ctools_content_configure_form_d
   // Unless we're not allowed to override the title on this content type, add this
   // gadget to all panes.
   if (empty($plugin['no title override']) && empty($subtype['no title override'])) {
-    $form['aligner_start'] = array(
+
+    $form['overrideTitle'] = array(
+      '#type'         => 'fieldset',
+      '#title'        => t('Reemplazar titulo'),
+      '#collapsible'  => TRUE,
+      '#collapsed'    => TRUE,
+    );
+
+    $form['overrideTitle']['aligner_start'] = array(
       '#markup' => '<div class="option-text-aligner clearfix">',
     );
-    $form['override_title'] = array(
+    $form['overrideTitle']['override_title'] = array(
       '#type' => 'checkbox',
       '#default_value' => isset($conf['override_title']) ? $conf['override_title'] : '',
-      '#title' => t('Override title'),
+      '#title' => t('Reemplazar titulo'),
+      // '#title' => t('Override title'),
       '#id' => 'override-title-checkbox',
     );
-    $form['override_title_text'] = array(
+    $form['overrideTitle']['override_title_text'] = array(
       '#type' => 'textfield',
       '#default_value' => isset($conf['override_title_text']) ? $conf['override_title_text'] : '',
       '#size' => 35,
@@ -508,7 +517,7 @@ function ctools_content_configure_form_d
       '#dependency' => array('override-title-checkbox' => array(1)),
       '#dependency_type' => 'hidden',
     );
-    $form['override_title_heading'] = array(
+    $form['overrideTitle']['override_title_heading'] = array(
       '#type' => 'select',
       '#default_value' => isset($conf['override_title_heading']) ? $conf['override_title_heading'] : 'h2',
       '#options' => array(
@@ -526,14 +535,16 @@ function ctools_content_configure_form_d
       '#dependency_type' => 'hidden',
     );
 
-    $form['aligner_stop'] = array(
+    $form['overrideTitle']['aligner_stop'] = array(
       '#markup' => '</div>',
     );
     if (is_array($contexts)) {
-      $form['override_title_markup'] = array(
+
+      $form['overrideTitle']['override_title_markup'] = array(
         '#prefix' => '<div class="description">',
         '#suffix' => '</div>',
-        '#markup' => t('You may use %keywords from contexts, as well as %title to contain the original title.'),
+        '#markup' => t('Puede usar %keywords de contextos, así como %title para contener el título original'),
+        // '#markup' => t('You may use %keywords from contexts, as well as %title to contain the original title.'),
       );
     }
     $add_submit = TRUE;
